.github/workflows/eks-ci.yml

```diff
@@ -65,7 +65,6 @@ jobs:
           android: true
           dotnet: true
           haskell: true
-          large-packages: true
```

Comment:
It defaults to `true`, just deleting it won't do anything

Reply:
Much faster. Let's see if the freed up space is enough
<img width="1318" alt="image" src="https://github.com/elastic/cloudbeat/assets/85433724/66560f4f-759a-4345-8639-c693250b9cdc">